api/kessel/inventory/v1beta1/relationships/k8spolicy_ispropagatedto_k8scluster.proto

```diff
@@ -10,6 +10,7 @@ import "buf/validate/validate.proto";
 option go_package = "github.com/project-kessel/inventory-api/api/kessel/inventory/v1beta1/relationships";
 option java_multiple_files = true;
 option java_package = "org.project_kessel.api.inventory.v1beta1.relationships";
+option java_outer_classname = "K8SPolicyIsPropagatedToK8SClusterProto";
```

Comment:
Do we need this on other definitions as well? or it fails for specific details on some of them?

Reply:
It's already been done in the past only requires K8s_ proto names.